lib/matplotlib/table.py

```diff
@@ -209,6 +248,14 @@ class Table(Artist):
 
     Column widths and row heights for the table can be specified.
 
+    The table consists of a grid of cells, which are indexed by (row, column).
+
+    For a simple table, you'll have a full grid of cells with indices from
+    (0, 0) to (max_rows-1, max_cols-1), in which the cell (0, 0) is positioned
```

Comment:
(n_rows-1, n_cols-1) or (max_row, max_col) (by definition max_foo is the largest value of foo...).